### PR TITLE
Multi footers

### PR DESCRIPTION
**Pull request by mirisuzanne**
hampster hat

## Description
- [Trello Card](https://trello.com/c/5RHOyGFG/24-i-want-to-swap-out-footer-call-to-action-blocks-per-page)
- [Staging Site](https://deploy-preview-14--oddleventy.netlify.com/) 

Allows us to create different "call-to-action" footers, and call them from any page. Current proof-of-concept uses the default footer from XD mockups, and the workshop footer from live oddsite (only on "services" page for now). This might require some adjustment…?

### CODE REVIEW

**Comment by mirisuzanne** [comment]
@SondraE Our overall font-sizes are a bit larger (which I tend to like), but I tried to split the difference and narrow the columns to a similar character-length. See what you think.

**Comment by SondraE** [comment]
@mirisuzanne 

- I like the changes you made to the footer content from my review. 
- I just noticed that the default dark blue link color on top of the light blue in the footer isn't quite high enough contrast.
Screen Shot 2020-01-06 at 12 16 24 PM

File: content/_sitedata/cta/default.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+templateEngineOverride: njk
+title: Want to chat about your web project or training?
+face: miriam
+summary: |
+  Contact us for custom web app design & development
+  with our team of Python, UX, and front-end experts --
+  or a 1-3 day workshop with Miriam Suzanne.
```
Comment (stacyk) [inline content/_sitedata/cta/default.md]: Is there any way to re-word this to not have so many horizontal lines in one small area?
Screen Shot 2020-01-08 at 3 38 02 PM

File: content/_sitedata/info.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  [Herman](/herman/) & [SassDoc](https://sassdoc.com/).\
+  See [sample pages](/sample/)
+  for basic usage.
+---
+
+{{ history | md | safe }}
+
+<p class="h-card">
+  Copyright © 2009&ndash;{% getDate 'year' %}
+  <a href="{{ site.url }}" class="p-name u-url">
```
Comment (stacyk) [inline content/_sitedata/info.md]: Is a homepage link here expected? I mean, I know that is how it is, but is this something people would know where they are going if they click it?

File: src/scss/layout/_cta.scss [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    0 calc(100% - var(--double-gutter))
+  );
+  display: grid;
+  font-size: var(--small);
+  grid-area: cta;
+  grid-template-columns: minmax(min-content, var(--small-page));
+  justify-content: center;
+  margin: 0 calc(var(--body-inline) * -1);
+  padding: var(--spacer) var(--body-inline) calc(var(--spacer) + var(--gutter));
+  text-shadow: 0 1px 0 var(--shadow);
```
Comment (stacyk) [inline src/scss/layout/_cta.scss]: hmmm, I only saw the text shadow when I selected text. Is that expected?

File: content/_sitedata/info.md (hunk 2 above)
Comment (mirisuzanne) [reply content/_sitedata/info.md]: Not sure it's important, just my habit to add the full company url inside our copyright. I learned that pattern a long time back for legal metadata, more than any particular user expectation. ¯\_(ツ)_/¯

File: src/scss/layout/_cta.scss (hunk 3 above)
Comment (mirisuzanne) [reply src/scss/layout/_cta.scss]: I removed it already in another branch…

File: content/_sitedata/cta/default.md (hunk 1 above)
Comment (mirisuzanne) [reply content/_sitedata/cta/default.md]: @SondraE I made a new trello card for considering this…

**Pull request merged**